<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>itemPricingSchedules</class>
 <widget class="QWidget" name="itemPricingSchedules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List Pricing Schedules</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="_searchForLit">
           <property name="text">
            <string>Search &amp;for:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>_searchFor</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="XLineEdit" name="_searchFor"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="Spacer6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="_showExpired">
         <property name="text">
          <string>Show E&amp;xpired Pricing Schedules</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_showFuture">
         <property name="text">
          <string>Show &amp;Future Pricing Schedules</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="_close">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer15">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="_itemPricingSchedulesLit">
       <property name="text">
        <string>&amp;Item Pricing Schedules:</string>
       </property>
       <property name="buddy">
        <cstring>_ipshead</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="XTreeWidget" name="_ipshead"/>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="_new">
             <property name="text">
              <string>&amp;New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_edit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_view">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;View</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_copy">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Copy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_delete">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_deleteExpired">
             <property name="text">
              <string>Delete Expired</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="Spacer17">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>120</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_searchFor</tabstop>
  <tabstop>_showExpired</tabstop>
  <tabstop>_showFuture</tabstop>
  <tabstop>_ipshead</tabstop>
  <tabstop>_new</tabstop>
  <tabstop>_edit</tabstop>
  <tabstop>_view</tabstop>
  <tabstop>_copy</tabstop>
  <tabstop>_delete</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <includes>
  <include location="global">parameter.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>itemPricingSchedules</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_ipshead</sender>
   <signal>valid(bool)</signal>
   <receiver>_view</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
